doc/source/user_guide/migration.rst

```diff
@@ -0,0 +1,215 @@
+.. _ref_migration_guide:
+
+Migration
+#########
+
+In this section two guides are provided to help you migrate from PyVista plotters to the Ansys Tools Visualization Interface plotters.
```

Comment:
```suggestion
This section helps you migrate from PyVista plotters to the Ansys Tools Visualization Interface plotters.
It consists of two major topics:

- Code migration
- Documentation migration
```
Make the two bullets links to the two sections given that they are fairly lengthy?